### PR TITLE
test for string_empty_to_null

### PR DESCRIPTION
**Pull request by 0xMALVEE**
Test written for string_empty_to_null.dart

image

### CODE REVIEW

File: cli/test/string_empty_to_null_test.dart [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import "package:stax/string_empty_to_null.dart";
+import "package:test/test.dart";
+
+void main() {
+  test("Test if empty string returns null", () {
```
Comment (TarasMazepa) [inline cli/test/string_empty_to_null_test.dart:5]: As soon as we already know this is a test, we do not need to have `"Test"` in the name of the test. Also let's just state description of test case without explicit result.  
```suggestion
  test("empty string", () {
```

File: cli/test/string_empty_to_null_test.dart [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    String emptyString = "";
+    expect(emptyString.emptyToNull(), null);
```
Comment (TarasMazepa) [inline cli/test/string_empty_to_null_test.dart:7]: You can inline this variable.
```suggestion
    expect("".emptyToNull(), null);
```

File: cli/test/string_empty_to_null_test.dart [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import "package:stax/string_empty_to_null.dart";
+import "package:test/test.dart";
+
+void main() {
+  test("Test if empty string returns null", () {
+    String emptyString = "";
```
Comment (TarasMazepa) [inline cli/test/string_empty_to_null_test.dart:6]: You can inline this variable.
```suggestion
```

File: cli/test/string_empty_to_null_test.dart [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    expect(emptyString.emptyToNull(), null);
+  });
+
+  test("Test if null string returns null", () {
```
Comment (TarasMazepa) [inline cli/test/string_empty_to_null_test.dart:10]:
```suggestion
  test("null", () {
```

File: cli/test/string_empty_to_null_test.dart [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    String? nullString;
```
Comment (TarasMazepa) [inline cli/test/string_empty_to_null_test.dart:11]: Let's use explicit initialization.
```suggestion
    String? nullString = null;
```

File: cli/test/string_empty_to_null_test.dart [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+    expect(nullString, null);
```
Comment (TarasMazepa) [inline cli/test/string_empty_to_null_test.dart:13]: You forgot to call `emptyToNull()` :) 
```suggestion
    expect(nullString.emptyToNull(), null);
```

**Review by TarasMazepa — CHANGES_REQUESTED** [review]
See comments. Also please add test case with not empty string.

**Review by TarasMazepa — APPROVED** [review]
Awesome!
